src/ApiNavigationList.vue

```diff
@@ -0,0 +1,72 @@
+<template>
+	<NcAppNavigationList :aria-labelledby="arialabel">
+		<NcAppNavigationItem v-for="app in apps"
+			:key="app.id"
+			allow-collapse
+			:name="app.name"
+			:open="isSelectedChild($route.params.appid, app)"
+			:to="{path: `/view/${app.id}`}">
+			<template #icon>
+				<AppIcon v-if="app.icon_url"
+					:href="app.icon_url" />
+				<ApiIcon v-else />
+			</template>
+			<NcAppNavigationItem v-for="child in childApis(app)"
```

Comment:
Since there are some predefined scopes (see https://github.com/nextcloud/server/blob/10c9fb926eea50e27eda6c29bb2047e7ba028148/lib/public/AppFramework/Http/Attribute/OpenAPI.php#L22) it would be nice to have some icons for the child specs as well.
That would be `administration`, `federation`, `ex_app` and `full` which you can just match and use a standard icon for.

Reply:
Don't know if we won't get involved into something bigger.
We can't control all the chosen names. See Talk:

![image](https://github.com/user-attachments/assets/2573f9ba-0db3-4500-8417-a3c43b6ada77)

Reply:
I know, but for the predefined ones we could have some icons.
In the end it's not that important, this is much better than we had before already.

Reply:
Nice idea!